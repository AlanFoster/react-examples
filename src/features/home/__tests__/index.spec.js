/* eslint-env browser */
import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../';

it('renders the home texts', () => {
  const component = renderer.create(<Home />);

  expect(component).toMatchSnapshot();
});
