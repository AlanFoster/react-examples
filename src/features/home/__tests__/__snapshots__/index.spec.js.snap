// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home texts 1`] = `
<div>
  Welcome to the home page!
</div>
`;
